packages/gasket-plugin-webpack/README.md

```diff
@@ -85,12 +70,51 @@ const config = initWebpack(gasket, webpackConfig, data);
 
 ## Lifecycles
 
+### webpackConfig
+
+Executed with initial, it receives three parameters:
```

Comment:
Not clear what "Executed with initial" means. Is it supposed to be `Executed by `initWebpack`?"